test/integration/scheduler/scheduler_test.go

```diff
@@ -269,4 +273,70 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			return createdJob.Spec.Suspend
 		}, framework.Timeout, framework.Interval).Should(gomega.Equal(pointer.Bool(false)))
 	})
+
+	ginkgo.It("Should schedule jobs according to the priority", func() {
+		// Initialize clusterQueue first but do not create
+		clusterQ := testing.MakeClusterQueue("cluster-queue").Resource(testing.MakeResource(corev1.ResourceCPU).
```

Comment:
I think we can use the existing ones (e.g., `prodQueue` defined above), otherwise we will need to add defer func to delete it.

Reply:
Done